src/lib.rs

```diff
@@ -253,7 +253,7 @@ where
         .context("failed to read descriptor set")?;
 
     let descriptor_set =
-        protobuf::parse_from_bytes::<descriptor::FileDescriptorSet>(&serialized_descriptor_set)
+        descriptor::FileDescriptorSet::parse_from_bytes(&serialized_descriptor_set)
```

Comment:
It might make sense to split this into a different commit, but I included it because I have compilation set up to fail when deprecated functions are used.